config/jobs/autobump/autobump.yaml

```diff
@@ -0,0 +1,59 @@
+periodics:
+- cron: "05 15 * * 1"  # Run at 15:05 PST (15:05 UTC) Mon
+- interval: 10m
+  name: ci-prow-autobump
+  decorate: true
+  extra_refs:
+  # Check out the repo containing the config and deployment files for your Prow instance.
+  - org: falcosecurity
+    repo: test-infra
+    base_ref: master
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/autobump # TODO: add a tag once we've push this the first time.
+      command:
+      - /autobump.sh
+      args:
+      - /etc/github-token/oauth
+      # Make the bot name and email match the user data of the provided token's user.
+      - "Falco Automation"
+      - jonah.jones094@gmail.com # change after tests work
```

Comment:
Is this to be merged ?

Reply:
O good catch, whats a good email to use for Falco?

Reply:
We could use @poiana's mail, no?